### ISSUE TITLE
recoil-sync import broken for NextJS app using .tsx

### ISSUE THREAD

**Issue by mikeebee** [facebookexperimental/Recoil#1775]
Hello! I just tried to use [recoil-sync](https://recoiljs.org/docs/recoil-sync/introduction) and I'm getting this error...

```
ReferenceError: exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/node_modules/recoil-sync/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```

I assume it's a similar thing to [#1773](https://github.com/facebookexperimental/Recoil/issues/1773)

Recoil (tried all `7x`)
Recoil-sync (tried `0.0.0` and `0.0.1-alpha.0`)
Next `12.1.0`
React `17.0.2`

I know this is only a preview branch currently, just wanted to make you aware 🙂

**Comment by mondaychen** [comment]
It should be fixed with 0.0.1-alpha.1

**Comment by mikeebee** [comment]
@mondaychen yes! It's loading now thank you, unfortunately, I'm seeing a new error regarding `window` not being available, probably due to SSR in NextJS.

```
index.js?46cb:352 Uncaught at getURL (file:///sites/node_modules/recoil-sync/cjs/index.js:2001:33)
```

I will create a separate issue for this.
